pkg/controller/masteruserrecord/masteruserrecord_controller.go

```diff
@@ -141,19 +139,19 @@ func (r *ReconcileMasterUserRecord) addFinalizer(logger logr.Logger, mur *toolch
 			return false, r.wrapErrorWithStatusUpdate(logger, mur, r.setStatusFailed(toolchainv1alpha1.MasterUserRecordUnableToAddFinalizerReason), err,
 				"failed while updating with added finalizer")
 		}
-		logger.Info("MasterUserRecord now has finalizer", "mur", mur)
+		logger.Info("MasterUserRecord now has finalizer")
 		return true, nil
 	} else {
-		logger.Info("MasterUserRecord already has finalizer", "mur", mur)
+		logger.Info("MasterUserRecord already has finalizer")
 	}
 	return false, nil
 }
 
-func (r *ReconcileMasterUserRecord) ensureUserAccount(logger logr.Logger, murAccount toolchainv1alpha1.UserAccountEmbedded, mur *toolchainv1alpha1.MasterUserRecord) error {
+func (r *ReconcileMasterUserRecord) ensureUserAccount(logger logr.Logger, murAccount toolchainv1alpha1.UserAccountEmbedded, mur *toolchainv1alpha1.MasterUserRecord) (bool, error) {
```

Comment:
And here as well. Please add a comment explaining what this method returns.

Reply:
done in 41d1d69